### ISSUE TITLE
Pendo should not initialize if a user is not logged in to zesty

### ISSUE THREAD

**Issue by giseleblair** [zesty-io/manager-ui#1310]
In the Manager, there is an initialization script that runs Pendo (3rd party tracking platform) which is initialized on every session of the manager. This should not run unless the user session is currently valid.

**Comment by giseleblair** [comment]
Pendo initialization script lives here: https://github.com/zesty-io/manager-ui/blob/master/src/shell/components/load-instance/index.js

**Comment by rjpals** [comment]
@giseleblair The pendo script will not initialize unless the user is logged in, as far as I can tell. There is [a check to prevent this](https://github.com/zesty-io/manager-ui/blob/master/src/shell/components/load-instance/index.js#L73-L79). 

However, you can have a situation where

1. a user logs in
2. Pendo gets initialized with that user's info
3. The user logs out
4. Pendo **remains initialized** with that user's info

Does this sound like a correct description of your issue? What side effects are you seeing (within Pendo) that show you that a user is not logged in but has Pendo initialized?

**Comment by giseleblair** [comment]
That is precisely the issue. When a session expires, the Pendo guides may still trigger. 

It looks like we can use the function `pendo.clearSession()` to end the current pendo session. This could be called when a zesty session expires. 

<img width="1440" alt="Screen Shot 2022-08-16 at 12 14 58 PM" src="https://user-images.githubusercontent.com/9896546/188036071-da1bf6e3-9d74-453b-93e5-fb53eab60d6b.png">

**Comment by giseleblair** [comment]
Pendo documentation: https://support.pendo.io/hc/en-us/articles/360039616892-Snippet-Installation-using-Signed-Metadata-with-JWT#can-i-use-signed-metadata-and-track-anonymous-visitors--0-7

**Comment by rjpals** [comment]
The current implementation in #1324 uses `pendo.clearSession()` -- I believe that should solve the issue.

It may be worth noting that this will create an "anonymous" user; Pendo is still initialized

**Comment by shrunyan** [comment]
Can we configure pendo to check whether a modal is open and only show announcements when it knows the screen is available?

**Comment by giseleblair** [comment]
Yes that is how it is set right now. So the user should not see this behavior again. But a specifically don't want Pendo to track this as an active session
